### ISSUE TITLE
Support template literals

### ISSUE THREAD

**Issue by mbenadda** [connectedcars/gettext-extractor#12]
Template literals are an integral part of modern JS (as well as TS). I have seen you use them in your codebase, yet they are not recognized as strings when using the extractor.

```typescript
// Not parsed
gt.gettext(`Starts on ${startsOn}`)
```

```
  0 messages extracted
  --------------------------
  0 total usages
  0 files (0 with messages)
  0 message contexts
```

```typescript
// Parsed
gt.gettext('Starts on ${startsOn}')
```

```
   1 message extracted
  -----------------------------
   1 total usage
  18 files (1 with messages)
   1 message context (default)
```

Obviously template literals are very valuable in translatable strings as they allow translators to move parameters around as needed in their locale.

In my opinion this is a critical feature to support. I'm open to working on a PR for this if needed.

**Comment by lukasgeiter** [comment]
I agree with that sentiment and I'll try to implement this in the coming days. It shouldn't be too difficult...

I have a question though, in your example, how would you actually retrieve the string containing `${startsOn}` in the `gettext` function during execution? Because normally you'd just get the string _after_ interpolation. So instead of `Starts on ${startsOn}` you'd get `Starts on <value-of-startsOn>`.

**Comment by mbenadda** [comment]
Hey Lukas,

You are correct, my suggested use case does not work as the string would indeed be impossible to retrieve. I planned to use [node-gettext](https://github.com/alexanderwallin/node-gettext) to retrieve strings and hoped to use template strings as node-gettext does not provide this functionality.

I'll just use lodash's `template` function instead. I guess you could still support template literals for completeness' sake but other users might then try the same broken approach I suggested.

Anyway, thanks for your answer and thanks for maintaining this package!

**Comment by lukasgeiter** [comment]
Support for template literals is now released under v3.3.0
